app/src/androidTest/java/com/example/henzoshimada/feeltrip/FeelTripTests.java

```diff
@@ -1,11 +1,11 @@
 package com.example.henzoshimada.feeltrip;
 
-import android.location.Location;
+import android.os.AsyncTask;
 import android.test.ActivityInstrumentationTestCase2;
-import android.util.Log;
```

Comment:
It's a Test Class, so we really don't need a LogCat for it. The Logcat is for the actual classes we're testing